addons/binding/org.openhab.binding.russoundserial/src/main/java/org/openhab/binding/russoundserial/handler/RussoundSerialBridgeHandler.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.russoundserial.handler;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.binding.BaseBridgeHandler;
+import org.eclipse.smarthome.core.types.Command;
+import com.thejholmes.russound.RussoundAction;
+import com.thejholmes.russound.RussoundCommander;
+import com.thejholmes.russound.RussoundZoneInfoListener;
+import com.thejholmes.russound.Zone;
+import com.thejholmes.russound.ZoneInfo;
+import com.thejholmes.russound.serial.SerialCommandReceiver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This does the actual communicating to the Russound device from the various Things that represent
+ * the zones.
+ *
+ * @author Jason Holmes - Initial contribution
+ */
+public class RussoundSerialBridgeHandler extends BaseBridgeHandler {
+    private final RussoundCommander commander;
+    private final SerialCommandReceiver commandReceiver;
+    private final ZoneListener zoneListener;
+
+    public RussoundSerialBridgeHandler(Bridge bridge, RussoundCommander commander,
+            SerialCommandReceiver commandReceiver, ZoneListener zoneListener) {
+        super(bridge);
+        this.commander = commander;
+        this.commandReceiver = commandReceiver;
+        this.zoneListener = zoneListener;
+    }
+
+    public RussoundCommander getCommander() {
+        return commander;
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
```

Comment:
At this point we've established we can make a serial connection, so setting it to `ONLINE` is fine here.